package.json

```diff
@@ -35,7 +35,7 @@
     "node-schedule": "1.2.0",
     "oas-tools": "^2.1.8",
     "uuid": "^3.3.2",
-    "vaultclient": "scality/vaultclient#21d03b1",
+    "vaultclient": "scality/vaultclient#ff9e92f",
```

Comment:
I know the PR that this commit hash relates to has been approved, but it might be better in the future if you/we all wait for a dependent PR to be merged before trying to use it in another project. As if that other commit had comments or failures the commit hash would change.

Reply:
okay, I usually ask for re-reviews.
Yes, if there are failures/changes the commit changes, even with squashes. 
Part of basic book keeping, when that happens I change it here too, I do take care of that.

Thats why had a couple of force pushes before asking for reviews.

Reply:
ok sounds right, just know it could potentially discourage review comments if it meant breaking another PR. Just a thought though.

Reply:
cool, yeah! if there are conflicts like these, its easier to do just one, like you suggested in your first comment.
This one was simple enough. But some need solo reviews as well, things which have more changes